packages/core/package.json

```diff
@@ -4,24 +4,17 @@
   "description": "UMA smart contracts and unit tests",
   "dependencies": {
     "@openzeppelin/contracts": "4.1.0",
-    "@truffle/contract": "^4.3.15",
     "@uma/common": "^2.7.0",
-    "@uma/core-1-1-0": "npm:@uma/core@1.1.0",
```

Comment:
No longer need older versions of core to support the index.ts file functions.

Reply:
!!